src/components/availability-panel/drag-drop-context/index.tsx

```diff
@@ -34,8 +34,13 @@ const DragDropcontext: FC<dragDropProps> = ({
   const [isTaskOnDrag, setIsTaskOnDrag] = useState<boolean>(false);
 
   useEffect(() => {
-    setTaskList(unAssignedTasks);
-    setMemberList(idleMembers);
+    const oldTasksIds = taskList.map((Task: task) => Task.id);
+    const newTasks = unAssignedTasks.filter((Task: task) => !oldTasksIds.includes(Task.id));
+    const newTaskList = [...taskList, ...newTasks];
```

Comment:
Let's move this logic to a function

Reply:
@akshay1502 Let's move this logic into a separate function